Libraries/LocalNotificationIOS/LocalNotificationIOS.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * @providesModule LocalNotificationIOS
+ */
+'use strict';
+
+var NativeModules = require('NativeModules');
+var RCTDeviceEventEmitter = require('RCTDeviceEventEmitter');
+var RCTLocalNotificationManager = NativeModules.LocalNotificationManager;
+var invariant = require('invariant');
+
+var _notificationHandlers = {};
+var _permissionHandlers = {};
+
+var DEVICE_LOCAL_NOTIFICATION_EVENT = 'localNotificationReceived';
+var DEVICE_PERMISSIONS_EVENT = 'notificationPermissionsReceived';
+
+/**
+ * Handles local notifications, including permissions, and icon badge number.
+ */
+var LocalNotificationIOS = {
+
+  /**
+   * Sets the badge number for the app icon
+   */
+  setApplicationIconBadgeNumber: function(number: number) {
```

Comment:
Yes, `getApplicationIconBadgeNumber` and `setApplicationIconBadgeNumber` are the same in both modules. However, I think they should be defined somewhere more globally because changing the badge number isn't limited to apps using notifications.